lib/src/utils/crypto/native.dart

```diff
@@ -78,6 +78,13 @@ abstract class Cipher {
       malloc.free(mem);
     }
   }
+  
+  Pointer<NativeType> getDigestName() {
+    final digestAlgo = messageDigestAlgorithm.toNativeUtf8();
+    final digestName = EVP_get_digestbyname(digestAlgo);
+    calloc.free(digestAlgo);
+    return digestName;
+  }
```

Comment:
why the commit related to `test` include the change to logic?

Reply:
i will rebase later. my bad